frontend/src/components/navigator/Layout.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Sidebar from "./Sidebar";
+import Navbar from "./Navbar";
+
+
+const Layout = ({ children }) => {
+  return (
+    <div style={{ display: "flex" }}>
+      <Navbar />
```

Comment:
diff b/w Navbar and Sidebar?

Reply:
Navbar is at the top whereas sidebar is the navigation on the side